script/Arcueid/Day 5/05_00_ARC05_1B.txt

```diff
@@ -291,17 +292,17 @@ Arcueid looks at me, furrowing her brow.
 [sha:cb7906128247e27a2e6038f5a6d604efe17bbe06]{
 -- Page 57, Offset 8731.
 -- 当然の反応だ。
-It's a natural reaction.
+A natural reaction.
 }
 [sha:62cdc41ced21b1ddd7113a930214dab0b6dee82a]{
 -- Page 57, Offset 8732.
 -- 加害者である俺が、アルクェイドを殺した理由にまったく見当がつかないっていうんだから。
-After all, with me being the perpetrator, there's no way that she could know the reason I killed her.
+After all, I, the perpetrator, just said I didn't know why I killed her.
 }
 [sha:1a58a543435e737145c36984709667fe3f6db8d3]{
 -- Page 58, Offset 8733.
 -- 「理由なんてないんじゃない？
-"There was no reason for it, right?
+"There wasn't any reason, right?
```

Comment:
Is this supposed to be "I mean, it's not like you needed a reason anyway right?" Makes more sense with next sentence

Reply:
yeah seems right